certify-mock-mdl.properties

```diff
@@ -19,6 +19,8 @@ mosip.certify.domain.url=https://${mosip.injicertify.mock.host}
 mosip.certify.identifier=${mosip.certify.domain.url}
 mosip.certify.authorization.url=https://${mosip.esignet.mock.host}
 mosip.certify.database.name=inji_certify_mock
+# conditionally set the SOFTHSM pin based on the setup requirements and fallback to the latter if former value is missing
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.certify.mockid.security.pin:${softhsm.certify.security.pin}}
```

Comment:
As of now, the mockid security PIN is going to be the same for all ENVs, later when we deploy me might plan to 

1. change the way how it's done
2. make certain exceptions to the rule to make the deployment simpler, i.e. deploy inji-certify-mock-mdl in the same namespace as inji-certify-mock pod


cc @abhishek8shankar @vishwa-vyom